### ISSUE TITLE
coreui Pro install npm

### ISSUE THREAD

**Issue by nallias** [coreui/coreui#17]
I ran into multiple troubles (withyour latest pro zip coreui-pro-bootstrap-admin-template-2.0.1.zip and  npm 6.1.0):

package.json asks for copyfile 2.0.1, which does not exist

```
"devDependencies": {
    [...]
    "copyfiles": "^2.0.1",
```

If I enforce copyfiles 2.0.0 ([as it is the latest existing one](https://www.npmjs.com/package/copyfiles)), 

```
"devDependencies": {
    [...]
    "copyfiles": "^2.0.0",

```
it works (NB: do not run as root and do not install globally)

Was there a confusion between coreui Pro 2.0.1 and copyfiles 2.0.0?

**Comment by xidedix** [comment]
Hi @nallias 
there are two different issues:
1. latest version of `copyfiles` is `2.0.0` actually - sorry for the inconvenience - you can fix it by adjusting your package.json in devDependencies
2. `node-sass` installation/permission issues - please try to clean up the npm files and reinstall:
```bash
rm -rf node_modules
rm package-lock.json
npm cache clean --force
npm install
```

**Comment by xidedix** [comment]
@nallias 
`copyfiles` version glitch has been fixed in the v2.1.0 release of coreui-pro-bootstrap-admin-template

**Comment by nallias** [comment]
Thanks for the fix xidedix 👍
